### ISSUE TITLE
timeout ignored for foreground starts

### ISSUE THREAD

**Issue by bdw429s** [cfmlprojects/runwar#58]
We noticed while testing CommandBox that when background is set to false, the timeout flag is ignored.  Just wanted to check if this is a bug or a desired behavior.  I assume this is the case because a process builder is not used to start the server in a controlled way but the server is just started directly in the main process.  Please confirm.

**Comment by denuno** [comment]
Unless I'm thinking of something else, this is desired behavior.  There should be no timeout for a foreground start, it should run until it hits a runtime error or gets a control-C.  AFAIK.

**Comment by bdw429s** [comment]
Yeah, that confirms what I was thinking.  Just wanted to make sure I was thinking correctly.
